nginx_conf.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Updating and installing required commands
+apt update  -y
```

Comment:
we don't need this, as if everything is updated in `nginx-install.sh` file

Reply:
done